src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly.js

```diff
@@ -48,6 +48,7 @@ const BaseAnchorForCommentsOnly = (props) => {
     return (
         <PressableWithSecondaryInteraction
             inline
+            style={[styles.dInline, styles.cursorDefault, {fontSize: props.style.fontSize}]}
```

Comment:
@parasharrajat I plan to add a comment here which says that the `inline` prop is only used for the native environment.